index.html

```diff
@@ -4,63 +4,219 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Figma Template 1</title>
-    <link rel="stylesheet"
-          href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
-    <link rel="stylesheet" href="style.css">   
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
+    <link rel="stylesheet" href="style.css">
+    <title>Leon's Portfolio</title>
 </head>
 <body>
-    <header>
-        <a class="logo">Leon Espitia</a>
-        <nav class="menu_icon">
-            <div>
-              <img src="./assets/icons/Hamburguer.svg" alt=""/>
-            </div>  
-        </nav>
-    </header>
-
     <main>
-        <section class="headline">
-          <h1 class="title">I'm Leon. Happy to see you!</h1>
-          <p class="headline_content">
-            I'm a software developer! I can help you build a product,
-            feature or website Look through some of my work and experience! 
-            If you like what you see and have a project you need coded, 
-            don't hestiate to contact me.
-          </p>
+<!--card_1 starts from here--> 
+        <div>
+            <section class="picture_container">
+                    <img src="./assets/pictures/picture1.jpg" alt=""/>
+                <div class="info_container">   
+                    <h1 class="title">Tonic</h1>
```

Comment:
- [ ] You should only one `<h1>` tag in an HTML document for semantic reasons and SEO. In this case, there is one `<h1>` in the headline section and you should use `<h2> ... <h6> `here.